panther_lights_scheduler/src/scheduler_node.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+
+import rospy
+
+from sensor_msgs.msg import BatteryState
+from std_msgs.msg import Bool
+
+from panther_msgs.msg import LEDAnimation
+from panther_msgs.srv import SetLEDAnimation, SetLEDAnimationRequest
+
+
+class LightsSchedulerNode:
+    def __init__(self, name) -> None:
+        rospy.init_node(name, anonymous=False)
+
+        self._battery_percentage = 1.0
+        self._charger_connected = False
+        self._e_stop_state = None
+        self._led_e_stop_state = None
+        self._last_battery_anim_time = 0.0
+
+        self._critical_battery_anim_period = rospy.get_param('critical_battery_anim_period', 15.0)
+        self._critical_battery_margin = rospy.get_param('critical_battery_margin', 0.1)
```

Comment:
```suggestion
        self._critical_battery_threshold_percent = rospy.get_param('critical_battery_threshold_percent', 0.1)
```
My first thought after reading “margin” was some boundary conditions of battery...? Threshold is not the best word for that case, either. Especially with the additional word being “percentage”, but for some reason “margin” in my opinion is strictly connected with something about boundaries.